automate.py

```diff
@@ -17,6 +17,9 @@ def __init__(self, tracer):
         self._tracer = tracer
         self._circuits = []
         self.find_circuits()
+        self.trigger = ""
```

Comment:
@Alopalao when you work on this comment https://github.com/kytos-ng/sdntrace_cp/pull/36/files#r967114548, I believe we can get rid of `self.trigger` and `self.kwagrs` here since you'd have it from `settings` that will be passed.

Reply:
Ok.